internal/distro/fedora/images.go

```diff
@@ -385,6 +385,16 @@ func iotRawImage(workload workload.Workload,
 	img.Users = users.UsersFromBP(customizations.GetUsers())
 	img.Groups = users.GroupsFromBP(customizations.GetGroups())
 
+	var err error
+	img.Directories, err = blueprint.DirectoryCustomizationsToFsNodeDirectories(customizations.GetDirectories())
```

Comment:
Same here for the generalization

Reply:
Commented in the Fedora distro. TL;DR; I'm not in favor of generalizing this.